File: CMS/src/app/shared/ATTPInfoProductBads.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ATTPInfoProductBads } from 'src/app/shared/ATTPInfoProductBads.model';
import { BaseService } from './Base.service';
@Injectable({
  providedIn: 'root'
})
export class ATTPInfoProductBadsService extends BaseService {
  constructor(public httpClient: HttpClient) {
    super(httpClient);
    this.Controller = "ATTPInfoProductBads";
  }
}

File: Service/Implement/ATTPInfoProductBadsService.cs
namespace Service.Implement
{
    public class ATTPInfoProductBadsService : BaseService<ATTPInfoProductBads, IATTPInfoProductBadsRepository>
    , IATTPInfoProductBadsService
    {
    private readonly IATTPInfoProductBadsRepository _ATTPInfoProductBadsRepository;
    public ATTPInfoProductBadsService(IATTPInfoProductBadsRepository ATTPInfoProductBadsRepository) : base(ATTPInfoProductBadsRepository)
    {
    _ATTPInfoProductBadsRepository = ATTPInfoProductBadsRepository;
    }
    }
    }

